test/library/packages/BLAS/BLAS.future

```diff
@@ -0,0 +1,2 @@
+chpl: /lus/scratch/killough/chapel/third-party/llvm/llvm-src/lib/IR/Instructions.cpp:2873: static llvm::CastInst* llvm::CastInst::Create(llvm::Instruction::CastOps, llvm::Value*, llvm::Type*, const llvm::Twine&, llvm::Instruction*): Assertion `castIsValid(op, S, Ty) && "Invalid cast!"' failed.
+timedexec: target program died with signal 6, with coredump
```

Comment:
It sounds like this PR gets LAPACK tests passing with LLVM but not BLAS? Or is this a leftover file?

Anyway I don't see what this .future would actually do in the testing system. There is no BLAS.chpl test in this directory and there aren't directory-wide futures.

Reply:
One of the LLVM BLAS tests failed with an error which seems unrelated to this PR. I'm not sure I used the right syntax for future files -- I just copied the failing output.

Reply:
Sure. Have a look at https://github.com/chapel-lang/chapel/blob/master/doc/rst/developer/bestPractices/TestSystem.rst#a-test-that-tracks-a-failure to better understand the .future files.